recipes/libssh2/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "libssh2" %}
+{% set version = "1.7.0" %}
+{% set sha256 = "a6de91a743c57c2d4168be9ce70f9a4ef5a58d12ffe7b0c0b393ed8f620f3f7a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+      - vc9    # [win and py27]
+      - vc10   # [win and py34]
+      - vc14   # [win and py35]
+
+requirements:
+  build:
+    - python         # [win]
+    - cmake >2.8
+    - openssl 1.0.*
```

Comment:
Sorry @jakirkham, can't say by looking at the documentation or the build output. Certificates, I don't even know what that means.